mani_skill/envs/tasks/drawing/draw_triangle.py

```diff
@@ -0,0 +1,330 @@
+import math
```

Comment:
![image](https://github.com/user-attachments/assets/7c4b8126-d962-444c-a1bd-6d3c3de18090)

Is it possible to make the outline 1. not transparent and 2. just be below the dots/ink? instead of over it. Looks a bit strange here